src/components/Executions/ExecutionDetails/test/ExecutionNodeViews.test.tsx

```diff
@@ -29,6 +29,8 @@ describe.skip('ExecutionNodeViews', () => {
         props = { execution: workflowExecution };
     });
 
+    it('is disabled', () => {});
```

Comment:
Are these intentionally kept in?

Reply:
Yep. This keeps the test runner from throwing errors. I will remove them in the next PR when I fix and re-enable the commented-out tests.